app/views/works/_work_header_navigation.html.erb

```diff
@@ -92,7 +92,7 @@
       <%= render 'subscriptions/form', :subscription => @subscription %>
     </li>
   <% end %>
-
+  <% if @work.posted?  %>
```

Comment:
It interesting in a way that we are optimising the indentation for the code rather than the html it produces.